crates/red_knot_python_semantic/src/types/builder.rs

```diff
@@ -258,6 +275,26 @@ mod tests {
         }
     }
 
+    fn setup_db_with_python() -> TestDb {
```

Comment:
I think it would be fine to just add all this to the existing `setup_db` in this module. It's using a memory "file system" so nothing very slow is happening here. If the builders may now depend on Program, it's better to just have it in all tests and not have to think about which tests need what kind of setup.

Reply:
Done!